### ISSUE TITLE
Missing dep: object.assign

### ISSUE THREAD

**Issue by wmertens** [reactjs/react-a11y#115]
See #113 - object.assign should be added as a dependency or included in the build.

**Comment by vitkon** [comment]
Module not found: Error: Cannot resolve module 'object.assign' in /private/var/www/sandbox2/common-accessible-components/node_modules/react-a11y/dist

**Comment by ljharb** [comment]
Thanks for the report - indeed object.assign needs to be moved from devDeps to deps.

**Comment by wyattdanger** [comment]
@wmertens @vitkon 0.3.1 is released

**Comment by wmertens** [comment]
:beer:
